portfolio/pom.xml

```diff
@@ -23,10 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
     <dependency>
         <groupId>com.google.code.gson</groupId>
         <artifactId>gson</artifactId>
         <version>2.8.6</version>
+
```

Comment:
hey carolyn, any reason for this extra line?

Reply:
Fixed!